homeassistant/components/auth/login_flow.py

```diff
@@ -199,23 +199,19 @@ async def get(self, request: web.Request) -> web.Response:
         )
 
 
-def _prepare_result_json(
-    result: AuthFlowResult,
-) -> AuthFlowResult:
-    """Convert result to JSON."""
+def _prepare_result_json(result: AuthFlowResult) -> dict[str, Any]:
```

Comment:
Can we make another typed dict that is correct instead of using `dict[str, Any]`?